pkg/deploy/generator/resources.go

```diff
@@ -122,6 +122,34 @@ func (g *generator) securityGroupPE() *arm.Resource {
 	}
 }
 
+func (g *generator) borderRouteTable() *arm.Resource {
+	return &arm.Resource{
+		Resource: &mgmtnetwork.RouteTable{
+			RouteTablePropertiesFormat: &mgmtnetwork.RouteTablePropertiesFormat{
+				Routes: &[]mgmtnetwork.Route{
+					{
+						Name: to.StringPtr("to-border"),
+						RoutePropertiesFormat: &mgmtnetwork.RoutePropertiesFormat{
+							AddressPrefix:    to.StringPtr("0.0.0.0/0"),
+							NextHopType:      mgmtnetwork.RouteNextHopTypeVirtualAppliance,
+							NextHopIPAddress: to.StringPtr("10.0.200.4"),
```

Comment:
@mjudeikis, to investigate this.

Reply:
@mjudeikis I have updated this but it fails to deploy since it does not exist as a separate object in the template.  I'm attempting to see if you can create a nic and then attach it to the VMSS.  I do not believe this is possible as a VMSS has a VMSS nic configuration.

Reply:
Updated.